src/nyc_trees/js/src/event.js

```diff
@@ -2,11 +2,11 @@
 
 var $ = require('jquery'),
     L = require('leaflet'),
-    mapModule = require('./map'),
+    EventMap = require('./EventMap'),
     fetchAndReplace = require('./fetchAndReplace');
 
 var $map = $('#map');
-mapModule.create({
+new EventMap({
```

Comment:
Using ``new`` without assigning the value to anything looks a little odd. Is there a reason to move away from the ``mapModule.create(...)`` style?